src/app/modules/submission/email-submission/ParsedResponsesObject.class.ts

```diff
@@ -152,7 +152,8 @@ export default class ParsedResponsesObject {
           // encrypt mode submissions with responses on unhidden fields
           // TODO(#780): Remove this once submission service is separated into
           // Email and Encrypted services
-          form.responseMode === FormResponseMode.Encrypt
+          form.responseMode === FormResponseMode.Encrypt &&
+          !form.get('newEncryptionBoundary')
```

Comment:
to add unit test for this

Reply:
I realised that this isn't used for the old storage mode submission endpoint anyway! So I removed the conditional in [0dbda94](https://github.com/opengovsg/FormSG/pull/6595/commits/0dbda9428cff48db6f3efe92efe663e6738d6334). I think this is what should be done with the TODO tagged anyway (#780)